projects/js-packages/charts/src/charts/line-chart/line-chart.tsx

```diff
@@ -488,9 +491,9 @@ const LineChartInternal = forwardRef< SingleChartRef, LineChartProps >(
 							onPointerOut={ onPointerOut }
 							pointerEventsDataKey="nearest"
 						>
-							<Grid columns={ false } numTicks={ 4 } />
-							<Axis { ...chartOptions.axis.x } />
-							<Axis { ...chartOptions.axis.y } />
+							{ gridVisibility !== 'none' && <Grid columns={ false } numTicks={ 4 } /> }
```

Comment:
Can the `gridVisibility` also updated in the [types for the line chart](https://github.com/Automattic/jetpack/blob/050774c7fdb1c321fc17ad9398f9af1a52a2d42f/projects/js-packages/charts/src/charts/line-chart/types.ts#L46)

Reply:
eg. https://github.com/Automattic/jetpack/blob/f4860732d1aaa7ada3d88018e327be7452f0c4bc/projects/js-packages/charts/src/charts/conversion-funnel-chart/types.ts#L59